File: templates/template.py
path = "C:/Users/Lars/Documents/Programming/aoc/inputs/{year}/day{day}.txt"

def parse_puzzle(data):
  return data

def puzzle_input():
  with open(path) as f:
    return parse_puzzle(f.read())

def puzzle1(data):
  pass

def puzzle2(data):
  pass


print(puzzle1(puzzle_input()))
print(puzzle2(puzzle_input()))

File: templates/template.clj
(ns aoc.year{year}.day{day}
  (:require [aoc.utils :as utils]))

(defn parse-input [input]
  input)

(defn puzzle-input [] (parse-input (utils/get-input {year} {day})))
(defn test-input [] (parse-input ""))

(defn puzzle1 [input]
  :todo)

(defn puzzle2 [input]
  :todo)

(comment
  ;;Room for tests
  (puzzle1 (test-input))
  (puzzle1 (puzzle-input));; Result
  ;;Room for tests
  (puzzle2 (test-input))
  (puzzle2 (puzzle-input));; Result
  )